frontend/libs/studio-components/src/test-utils/testRootClassNameAppending.ts

```diff
@@ -9,9 +9,11 @@ export function testRootClassNameAppending(
   const defaultClasses: string[] = getRootClasses(defaultContainer);
   const { container: containerWithCLass } = renderComponent(className);
   const rootElement: HTMLElement = getRootElementFromContainer(containerWithCLass);
-  defaultClasses.forEach((defaultClass) => {
-    expect(rootElement).toHaveClass(defaultClass);
-  });
+  defaultClasses
+    .filter((c) => !!c)
```

Comment:
What about using the built in Boolean here?
```suggestion
    .filter(Boolean)
```

Reply:
Excellent idea, thanks!